tests/test_formatters.py

```
@@ -206,524 +206,526 @@
+    with pytest.raises(ValueError):
+        format_day(0, order='YYY')
+    with pytest.raises(ValueError):
+        format_day(0, ydigits=3)
+    with pytest.raises(ValueError):
+        format_day(0, kind='X')
+    with pytest.raises(ValueError):
+        format_day(0, buffer=np.empty((3,), dtype='int'))
+    with pytest.raises(ValueError):
+        format_day([[0,0],[0,0]], buffer=np.empty((3,3), dtype='U40'))
+
+def test_format_sec():
```

Ruff (0.15.6)

[warning] Too many branches (17 > 12)

(PLR0912)